renderdoc/driver/dxgi/dxgi_common.cpp

```diff
@@ -1391,7 +1391,8 @@ DXGI_FORMAT MakeDXGIFormat(ResourceFormat fmt)
         break;
       case eSpecial_S8: RDCERR("D3D has no stencil-only format"); break;
       case eSpecial_D16S8: RDCERR("D3D has no D16S8 format"); break;
-      default: RDCERR("Unrecognised/unsupported special format %u", fmt.specialFormat); break;
+      default:
+        break;    // Return DXGI_FORMAT_UNKNOWN to convert texture.
```

Comment:
There are still other RDCERR() calls in this function which would need to return DXGI_FORMAT_UNKNOWN as they correspon to valid formats in other APIs - e.g. for three-component 8-bit and 16-bit formats (which D3D doesn't support).